test/generators/github/store/active_record_generator_test.rb

```diff
@@ -5,15 +5,20 @@
 require "rails/generators/test_case"
 require "generators/github/ds/active_record_generator"
 
-class GithubKVActiveRecordGeneratorTest < Rails::Generators::TestCase
-  tests Github::DS::Generators::ActiveRecordGenerator
+class GithubDSActiveRecordGeneratorTest < Rails::Generators::TestCase
```

Comment:
Missed this and line below in https://github.com/github/github-ds/pull/16 so snagging it here.